cli/template/base/_eslintrc.cjs

```diff
@@ -6,7 +6,7 @@ const config = {
   },
   plugins: ["@typescript-eslint"],
   extends: [
-    "next/core-web-vitals",
+    "plugin:@next/next/recommended",
```

Comment:
[Looks like this change makes checks less strict](https://github.com/vercel/next.js/blob/canary/packages/eslint-plugin-next/src/index.ts#L58-L59) by removing errors for:

* `'@next/next/no-html-link-for-pages': 'error',`
* `'@next/next/no-sync-scripts': 'error',`

Out of curiosity - why did you decide to change this? (Asking since I'm genuinely curious 🙏)